pkg/cmd/backuprestore/backupserver.go

```diff
@@ -34,7 +34,6 @@ func (b backupRunnerImpl) runBackup(backupOpts *backupOptions, pruneOpts *prune.
 		return err
 	}
 
-	pruneOpts.BackupPath = backupVolume + "/" + dateString
```

Comment:
now that I'm reading this again, `paths.Join` would also be a safer option

Reply:
yes u r right, should have written test :'( 

fixing ...

Reply:
updated 👍🏽